lib/Modules/ComputeHooks/ComputeHooks.psm1

```diff
@@ -106,1360 +56,800 @@ function Install-Prerequisites {
     return $rebootNeeded
 }
 
-function Get-OpenstackVersion {
-    $distro = Get-JujuCharmConfig -Scope "openstack-version"
-    if($distro -eq $false){
-        $distro = Get-JujuCharmConfig -Scope "openstack-origin"
-    }
-    return $distro
-}
+function New-ExeServiceWrapper {
+    $pythonDir = Get-PythonDir -InstallDir $NOVA_INSTALL_DIR
+    $python = Join-Path $pythonDir "python.exe"
+    $updateWrapper = Join-Path $pythonDir "Scripts\UpdateWrappers.py"
 
-function Get-NetType {
-    $net_type = Get-JujuCharmConfig -Scope "network-type"
+    $cmd = @($python, $updateWrapper, "nova-compute = nova.cmd.compute:main")
+    Invoke-JujuCommand -Command $cmd
 
-    $distro = Get-OpenstackVersion
-    if($distro_urls["ovs"] -eq $false -or (Get-IsNanoServer)){
-        #force hyperv network manager for versions that do now support ovs
-        $net_type = "hyperv"
+    $version = Get-OpenstackVersion
+    $consoleScript = "neutron-hyperv-agent = neutron.cmd.eventlet.plugins.hyperv_neutron_agent:main"
+    if ($version -ge "mitaka") {
```

Comment:
I know it works, but comparing strings with a greater-or-equal test seems wrong :). It assumes the naming scheme will never change and will never come back to the letter M. We will never support more then 2 releases, but this still feels wrong :). No need to change if you are confident the test works as expected. Its more of a NIT that makes my left eye twitch when I see it.